Bug: Disabling notification alert toggle

MTSaeed opened the pull request:
**What does this PR?**
Keeps the drop down open after toggling for the notification settings.
By default we set the value of menuHide to true. if the value coming from porps is false, it keeps the drop down open. otherwise it closes the drop down after click on item.

**Steps to reproduce**

1. Open the notifications and then open then hover on setting, it will open the drop down.
2. Toggle different options. it will keep open the drop down.

**Issue number** 

fixes #1202

**Screenshot / Video**


https://user-images.githubusercontent.com/106739598/220539602-bd70eedd-dfe9-469f-a4d2-12e7fc7fc52a.mp4

feruzm commented:
@MTSaeed Can you check if after reloading or restarting browser, you will see same settings fetched correctly on your browser and different other browsers? Also please check if switching accounts does keep/fetch settings correctly on notifications alert settings.

MTSaeed commented:
> @MTSaeed Can you check if after reloading or restarting browser, you will see same settings fetched correctly on your browser and different other browsers? Also please check if switching accounts does keep/fetch settings correctly on notifications alert settings.

Yes. Everything is working fine. Restarting / Reloading browser or switching to different accounts keeps the settings correctly.

feruzm commented:
@MTSaeed Great, note by keep settings in you mean, settings saved for that account is fetched correctly, right? Each account has different settings and it is stored on backend.

MTSaeed commented:
@feruzm You can check in this video. i saved the different notification settings for demo.com and hive-189310. I switched the account and as well as i changed the browser, the account keeps setting correctly.

https://user-images.githubusercontent.com/106739598/220632669-e3c0caee-beeb-4401-b439-25cae47234b3.mp4

feruzm reviewed (approved):
lgtm
